lib/generators/logidze/install/templates/migration.rb.erb

```diff
@@ -16,17 +16,26 @@ class <%= @migration_class_name %> < ActiveRecord::Migration
     end
 
     execute <<-SQL
-      CREATE OR REPLACE FUNCTION logidze_version(v bigint, data jsonb) RETURNS jsonb AS $body$
+      CREATE OR REPLACE FUNCTION logidze_version(v bigint, data jsonb, blacklist text DEFAULT '', whitelist text DEFAULT '') RETURNS jsonb AS $body$
         DECLARE
           buf jsonb;
+          filtered_data jsonb;
+          filtered_array jsonb;
         BEGIN
+          IF blacklist <> 'null' THEN
+            filtered_array := logidze_exclude_keys(data, array['log_data']::text[] || string_to_array(blacklist, ','));
+          ELSIF blacklist = 'null' AND whitelist <> 'null' THEN
+            filtered_array := logidze_include_keys(data, string_to_array(whitelist, ','));
```

Comment:
I'd prefer to leave the only one function for filtering, `logidze_exclude_keys`.
Then we have to calculate required keys beforehand, smth like:

```
keys_to_exclude := select array(select unnest(jsonb_object_keys(data)) except select unnest(whitelist));
filtered_array := logidze_exclude_keys(data, keys_to_exclude);
```